CMakeLists.txt

```diff
@@ -32,6 +32,34 @@ if (CONFIG_LV_ATTRIBUTE_FAST_MEM_USE_IRAM)
     target_compile_definitions(${COMPONENT_LIB} PUBLIC "-DLV_ATTRIBUTE_FAST_MEM=IRAM_ATTR")
 endif()
 
+elseif(ZEPHYR_BASE)
+
+if(CONFIG_LVGL)
+
+set(ZEPHYR_CURRENT_LIBRARY lvgl)
+include_directories(${ZEPHYR_BASE}/lib/gui/lvgl)
+
+target_include_directories(lvgl INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
is `lvgl` target defined somewhere else?

Reply:
Yes the `lvgl` target is defined on the zephyr side  (`lib/gui/lvgl/CMakeLists.txt`)